package.nls.json

```diff
@@ -94,6 +94,7 @@
 "mssql.intelliSense.lowerCaseSuggestions":"Should IntelliSense suggestions be lowercase",
 "mssql.persistQueryResultTabs":"Should query result selections and scroll positions be saved when switching tabs (may impact performance)",
 "mssql.queryHistoryLimit":"Number of query history entries to show in the Query History view",
+"mssql.createAzureFunction":"Create Azure Function…",

```

Comment:
`Create Azure Function with SQL Binding...`?

@dzsquared

Reply:
yes, including the SQL binding mention is important context for the user
"Create Azure Function with SQL Binding"
or
"Generate Azure Function with SQL Binding"